modules/analysis-common/src/main/java/org/elasticsearch/analysis/common/MultiplexerTokenFilterFactory.java

```diff
@@ -54,6 +60,22 @@ public TokenStream create(TokenStream tokenStream) {
         throw new UnsupportedOperationException("TokenFilterFactory.getChainAwareTokenFilterFactory() must be called first");
     }
 
+    @Override
+    public TokenFilterFactory getSynonymFilter(boolean lenient) {
+        if (lenient) {
+            return IDENTITY_FILTER;
```

Comment:
I have mixed feelings for this one, this will silently ignore all input that would be changed by the multiplexer. Is there a use case to set a `multiplexer` before a synonym filter ?